internal/controllers/utils/utils.go

```diff
@@ -1173,10 +1173,8 @@ func ClearPRCallbackAnnotations(object client.Object) {
 		return
 	}
 
-	// Remove all callback-related annotations
+	// Only clears CallbackReceivedAnnotation
```

Comment:
I think we either need to rename this function to `ClearPRCallbackReceivedAnnotation` or define a new function that implements this specialized clearing functionality.

Reply:
It has been renamed to ClearPRCallbackReceivedAnnotation.